templates/mopro-example-app/web/README.md

```diff
@@ -0,0 +1,25 @@
+# Web Prover
+
+This is used to benchmark proving time in browser with `snarkjs`.
+See: https://mopro-webprover.pages.dev/
+
+Current proofs:
```

Comment:
I guess we'd want this to be just one circuit to map to other ones? But this works too to start with